app/templates/courses/teacher-class-view.jade

```diff
@@ -350,10 +350,11 @@ mixin bulkAssignControls
       span :
     select.bulk-course-select.form-control
       each course in _.rest(view.latestReleasedCourses)
-        option(value=course.id selected=(course===state.get('selectedCourse')))
-          = i18n(course.attributes, 'name')
-          if !view.availableCourseMap[course.id]
-            = " " + translate('teacher.paren_new')
+        if classroom.capitalLanguage != 'C++' || (course.id === view.utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE || course.id === view.utils.courseIDs.COMPUTER_SCIENCE_2 || course.id === view.utils.courseIDs.COMPUTER_SCIENCE_3)
```

Comment:
I'm not sure `classroom.capitalLanguage` will ever be set to `'C++'` if the following doesn't have a `C++` version: https://github.com/codecombat/ozaria/blob/2b5f6477251f0ead66751179c38cec74ffec71e2/app/models/Classroom.coffee#L30

But haven't looked too closely.

Reply:
actually capitalLanguage is the only vairable i found in this template contains 'c++' or 'cpp' 😂, that's why i use this variable. is classroom has other keys like 'codeLanguage' or 'language' i can use?

Reply:
Yeah. The classroom has an `aceConfig` field that you can access like so: @classroom.get('aceConfig').language from the coffeescript view.

I believe the capitalLanguage gets initialized here: https://github.com/codecombat/codecombat/blob/6ac854c40a6ba198df5ab32e504b0ef19fa139d5/app/collections/Classrooms.coffee#L11

It is probably a good idea to update the utils method to handle C++ as well.

Reply:
@AndrewJakubowicz oh,yes, i look into classrooms.coffee, it is interesting for me to find that we use classroom.get('aceConfig').language multiple times but never assign it with a global vairable. By the way, noticed codecombat has cpp when update capitalizelanguage, oziria doesn't.
i'll update it in coffeescript view now.